fred/libraries/createembed.py

```diff
@@ -271,10 +271,10 @@ def issue_comment(data: dict) -> nextcord.Embed:
 
 
 def _single_mod_embed(mod: dict) -> nextcord.Embed:
-    if zulu_time := mod.get("last_version_date"):
+    if zulu_time := mod.get("last_version_date") and len(mod.get("versions")) > 0:
         ts = timestamp(f"{zulu_time[:19]}+00:00")
     else:
-        ts = "(New)"
+        ts = ""
```

Comment:
why the removal of `New`? is there a better phrase we could use than empty string?

Reply:
See in the lines below where `ts` is used